<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="trajectory_ranker_param_path" default="$(find-pkg-share autoware_trajectory_ranker)/config/simple_trajectory_ranker.param.yaml"/>
  <arg name="input_trajectories" default="~/input/trajectories"/>
  <arg name="output_trajectories" default="~/output/trajectories"/>

  <node pkg="autoware_trajectory_ranker" exec="simple_trajectory_ranker" name="trajectory_ranker_node" output="screen">
    <param from="$(var trajectory_ranker_param_path)" allow_substs="true"/>
    <param from="$(find-pkg-share autoware_trajectory_ranker)/config/metrics/trajectory_consistency.param.yaml"/>

    <remap from="~/input/candidate_trajectories" to="$(var input_trajectories)"/>
    <remap from="~/output/scored_trajectories" to="$(var output_trajectories)"/>
  </node>
</launch>
